### PR TITLE
build: fix nightly sqllogictest

### PR DESCRIPTION
**Pull request by madelynnblue**
The config variable got removed previously. Hard code it.

Release note: None

### CODE REVIEW

File: build/teamcity-sqllogictest.sh [hunk 1]
```diff
@@ -16,13 +16,13 @@ export BUILDER_HIDE_GOPATH_SRC=0
...
+  | tee "artifacts/logictest.log" \
   | go-test-teamcity
 
 # Need to specify the flex-types flag in order to skip past variations that have
 # numeric typing differences.
 build/builder.sh \
   stdbuf -oL -eL \
   make test TESTFLAGS="-v -bigtest -config local-opt,fakedist-opt -flex-types" TESTTIMEOUT='24h' PKG='./pkg/sql/logictest' TESTS='^TestSqlLiteCorrelatedLogic$$' 2>&1 \
-  | tee "artifacts/${config}.log" \
+  | tee "artifacts/logictest.log" \
```
Comment (jordanlewis) [inline build/teamcity-sqllogictest.sh]: Does this overwrite the file above? Seems like it needs to go to a different filename.

Comment (mjibson) [reply build/teamcity-sqllogictest.sh]: Derp yes. Fixed.

**Review by jordanlewis — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/39098#-:-LkjYG25DWwEqcHpMfNW:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @jordanlewis)

**Pull request merged**